src/gmp.c

```diff
@@ -21928,6 +22001,15 @@ gmp_xml_handle_end_element (/* unused */ GMarkupParseContext* context,
                       }
                     log_event_fail ("task", "Task", NULL, "created");
                     break;
+                  case 3:
```

Comment:
Does `copy_task` have a return value of 3 or should the return of `request_delete_task` be checked instead ?

Reply:
I've removed the case now because it does not apply to the return value that is actually checked as you said.
I do not want to change the behavior for now but we can discuss this separately later.